com.sap.cloud.lm.sl.common/src/main/java/com/sap/cloud/lm/sl/common/util/ListUtil.java

```diff
@@ -53,4 +53,12 @@
         return new ArrayList<T>(new LinkedHashSet<T>(list));
     }
 
+    public static <T> void addUnique(List<T> original, List<T> newValues) {
```

Comment:
Could it be achieved using Set<T>? - it will be more efficient. :)

Reply:
No, because if there are already duplicating values in the original list, we don't want to remove them.

Reply:
I don't like this method as well, so I've reconstructed the dependant logic not to depend on this. Closing this PR.